src/unix/signal.c

```diff
@@ -351,7 +351,7 @@ static int uv__signal_start(uv_signal_t* handle,
                             uv_signal_cb signal_cb,
                             int signum,
                             int oneshot) {
-  sigset_t saved_sigmask;
+  sigset_t saved_sigmask = 0;
```

Comment:
style nit: separate declaration from initialization.

Reply:
this will cause another useless line `saved_sigmask = 0;` I don't think it's a good practice.

Reply:
I don't disagree but it's the style currently used in most of the code in the project.

Reply:
I assume that `sigemptyset(&saved_sigmask);` is the right way to clear this anyway?